Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Docker Image 
+FROM node:14-alpine as build-stage 
+
+# Directory within the virtualizied Docker environment
+WORKDIR /usr/src/app
+
+# Copies package.json and yarn.lock to Docker environment
+COPY package.json yarn.lock ./
+
+#Install all node packages
+RUN yarn install
+
+# Copies everything over to the Docker environment
+COPY . .
+
+ENV REACT_APP_SERVER_API_URL=http://scrumlr.local/api
+ENV REACT_APP_LIVEQUERY_URL=ws://scrumlr.local/ws
+ENV SKIP_PREFLIGHT_CHECK=true
+
+RUN yarn build
```

Comment:
I would personally prefer to use the files of the build on the local machine for this image, because it should be way faster to create this image and also the build could be reused in the CI/CD process. But I'm fine with it either way

Reply:
One of the next steps will be that we will build the images in our CI/CD anyway, so the Dockerfile is already prepared for that